Vindicta.Altis/Undercover/undercoverMonitor.sqf

```diff
@@ -344,6 +342,21 @@ CLASS("UndercoverMonitor", "MessageReceiver");
 
 								if (animationState _unit in g_UM_undercoverAnims) exitWith { _suspicionArr pushBack [-1, "Surrender"]; _hintKeys pushback HK_SURRENDER; }; // Hotfix for ACE surrendering
 
+								// suspiciousness for specific animations
+								//if (animationState _unit == "acts_carfixingwheel") then {
+									//_suspicionArr pushBack [0.6, "Removing wheel from enemy vehicle?"]; _hintKeys pushback HK_ILLEGAL;
+								//};
+								
+								// disallow player using morphine on enemies 
+								if (animationState _unit == "ainvpknlmstpsnonwnondnon_medic1" || animationState _unit == "ainvppnemstpslaywnondnon_medicother" || animationState _unit == "ainvpknlmstpslaywnondnon_medicother") exitWith {
+									pr _nearUnits = nearestObjects [_unit, ["Man"], 12];
+									if (count _nearUnits > 1) then {
+										if (((side (_nearUnits#1)) != (side _unit)) && (side (_nearUnits#1)) != civilian) then {
```

Comment:
use `side group _unit` instead (and side group man, etc, for checking side of any unit in general). Because setCaptive makes your unit's side civilian, but your group's side stays the same.